### PR TITLE
Remove ActiveStorage file transfer in favor of a transfer directory

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/services/update_stacks_files_service.rb [hunk 1]
```diff
@@ -34,31 +33,28 @@ def delete!
...
     raise RequestError, 'Files in file_uploads not in cocina object'
   end
 
-  def check_signed_ids
-    return if file_uploads_map.values.all? { |signed_id| ActiveStorage.verifier.valid_message?(signed_id) }
-
-    raise RequestError, "Invalid signed ids found"
-  end
-
   # Copy the files from ActiveStorage to the Stacks directory
```
Comment (justinlittman) [inline app/services/update_stacks_files_service.rb:46]: Comment needs update.

File: app/services/update_stacks_files_service.rb [hunk 2]
```diff
@@ -34,31 +33,28 @@ def delete!
...
   def shelve_files
     file_uploads_map.each do |filename, signed_id|
```
Comment (justinlittman) [inline app/services/update_stacks_files_service.rb:48]: Can `signed_id` be renamed?

**Review by justinlittman — APPROVED** [review]
Approved pending consideration of comments.

File: app/services/update_stacks_files_service.rb (hunk 1 above)
Comment (jcoyne) [reply app/services/update_stacks_files_service.rb]: done

File: app/services/update_stacks_files_service.rb (hunk 2 above)
Comment (jcoyne) [reply app/services/update_stacks_files_service.rb]: done

**Pull request merged**